routers/repo/attachment.go

```diff
@@ -141,6 +142,15 @@ func GetAttachment(ctx *context.Context) {
 	//If we have matched and access to release or issue
 	fr, err := storage.Attachments.Open(attach.RelativePath())
 	if err != nil {
+		// If the file is missing from the disk, it will return 404 and remove the db record.
+		if os.IsNotExist(err) {
+			log.Error("Attachment data is missing from the disk [uuid: %v, path: %v]", attach.UUID, attach.RelativePath())
+			if err2 := models.DeleteAttachment(attach, false); err2 != nil {
```

Comment:
No, we SHOULD not do that. Maybe it's a configuration problem.

Reply:
I see, I gorgot it.